packages/hsl-shared-components/src/Button/Button.js

```diff
@@ -35,57 +34,64 @@ function size(kind, primary, small) {
   return small ? map.small : map.default;
 }
 
-
-const TouchableText = styled(({ primary, secondary, disabled, small, ...rest }) => (
```

Comment:
We don't want to pass all these props `Text`, because then we get warnings in console
```
warning.js?8a56:35 Warning: Unknown props `primary`, `success`, `secondary`, `transparent`, `rounded` on <div> tag. Remove these props from the element. For details, see https://fb.me/react-unknown-prop
```